definitions/ext-hcp_vault/definition.yml

```diff
@@ -0,0 +1,26 @@
+domain: EXT
+type: HCP_VAULT
+
+synthesis:
+  rules:
+    # telemetry from prometheus exporter
+  - identifier: instance

```

Comment:
Hi,
I just wanted to double check "how unique" this identifier is ?
Is it globally unique ? That would be the simplest case.

In case it's possible to make it more unique via [a composite identifier](https://github.com/newrelic/entity-definitions/blob/main/docs/synthesis.md), I would recommend doing that.
Thanks !
Thanks.

Reply:
The tag  `cluster` made me ask this question.

Reply:
Hey Andre-NR, the identifier already contains the cluster details, added the cluster itself as part of a TAG just to provide a better decoration for this entity